staging/src/kubevirt.io/client-go/api/v1/types.go

```diff
@@ -268,6 +268,10 @@ type PersistentVolumeClaimInfo struct {
 	// Preallocated indicates if the PVC's storage is preallocated or not
 	// +optional
 	Preallocated bool `json:"preallocated,omitempty"`
+
+	// Percentage of filesystem's size to be reserved when resizing the PVC
+	// +optional
+	FilesystemOverhead *cdiv1.Percent `json:"filesystemOverhead,omitempty"`
```

Comment:
out of curiosity, why isn't this a float instead of a string value representing a float?

Reply:
[Can't have a float type](https://github.com/kubernetes-sigs/controller-tools/issues/245)